### ISSUE TITLE
no dependencies in package.json in npm-package v1.5.5

### ISSUE THREAD

**Issue by oweitman** [ioBroker/adapter-react#10]
https://github.com/ioBroker/create-adapter/issues/657

this is the content of package.json in v1.5.5
```
{
  "name": "@iobroker/adapter-react",
  "version": "1.5.5",
  "description": "React classes to develop admin interfaces for ioBroker with react.",
  "author": {
    "name": "bluefox",
    "email": "dogafox@gmail.com"
  },
  "scripts": {},
  "repository": {
    "type": "git",
    "url": "git+https://github.com/ioBroker/adapter-react.git"
  },
  "keywords": [
    "iobroker",
    "adapter",
    "iot"
  ],
  "license": "MIT",
  "bugs": {
    "url": "https://github.com/ioBroker/adapter-react/issues"
  },
  "homepage": "https://github.com/ioBroker/adapter-react#readme",
  "devDependencies": {},
  "dependencies": {}
}
```

**Comment by AlCalzone** [comment]
The readme mentions that these dependencies should be added to the `devDependencies` of adapter which use this:
```
"@material-ui/core": "^4.11.2",
"@sentry/browser": "^5.29.2",
"@iobroker/adapter-react": "^1.5.5",
"gulp": "^4.0.2",
"del": "^6.0.0"
```
However, that is not necessary and leads to potential duplication and version mismatches. Instead, they should be in the **`dependencies`** of adapter-react, so they get installed when adding adapter-react as a devDependency.

**Comment by AlCalzone** [comment]
@GermanBluefox

**Comment by GermanBluefox** [comment]
> "@iobroker/adapter-react": "^1.5.5",

How adapter-react could be in dependencies of its own?

**Comment by oweitman** [comment]
Ok, unthinkable in this issue we address 2 separate problems.

1. as already noted, the actual package on npmjs has a package.Json with empty dependencies. Please publish a new version which includes the package.Json shown in the GitHub repository
2. the second problem is, the sentry dependencies is in this package in the devdependencies section. If the create Adapter issues the npm install command, the needed devdependency isn’t installed automatically
 
 For this Problem we have 2 solutions:
The create Adapter Skript issues the npm install command with the —dev Option, That the devdependencies is also going to be installed
But in this case maybe more dev packages from other packages were also installed
Or
Add the sentry dependency to the dependency section

**Comment by AlCalzone** [comment]
> > "@iobroker/adapter-react": "^1.5.5",
> 
> 
> 
> How adapter-react could be in dependencies of its own?

Nah that should not be

**Comment by oweitman** [comment]
i checked the content of new version 1.6, downloaded with
https://registry.npmjs.org/@iobroker/adapter-react/-/adapter-react-1.6.0.tgz
the dependencies in the package.json is again empty.

**Comment by AlCalzone** [comment]
> The create Adapter Skript issues the npm install command with the —dev Option, That the devdependencies is also going to be installed

Running `npm install`only causes the direct `devDependencies` of the generated adapter to be installed, not the `devDependencies` of the `devDependencies`. So we would need the necessary modules to be included in `adapter-react`'s `dependencies`.

**Comment by UncleSamSwiss** [comment]
IMHO this can be closed as it was fixed in 92edafa
